services/grpc/grpcService.js

```diff
@@ -145,32 +155,42 @@ class GrpcService extends EventEmitter {
    */
   subscribe() {
     // this.subscriptions['something'] = this.service.subscribeToSomething()
+    // super.subscribe()
```

Comment:
Is this a leftover?

Reply:
No, this is meant as a sample to show what subclasses should do in their implementation of this method - add stuff to `this.subscriptions` and call `super.subscribe`